web/src/lib/converter/document-session.svelte.ts

```
@@ -0,0 +1,297 @@
+			this.status = 'Could not open document';
+		} finally {
+			if (generation === this.loadGeneration) this.parsing = false;
+		}
+	}
+
+	async close(): Promise<void> {
+		const generation = ++this.loadGeneration;
+		try {
+			await this.requireClient().dispose(generation);
+		} finally {
+			if (generation !== this.loadGeneration) return;
```

Biome (2.5.8)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)